src/sentry/incidents/subscription_processor.py

```diff
@@ -562,7 +560,10 @@ def process_update(self, subscription_update: QuerySubscriptionUpdate) -> None:
                         if self.has_anomaly(
                             potential_anomaly, trigger.label
                         ) and not self.check_trigger_matches_status(trigger, TriggerStatus.ACTIVE):
-                            metrics.incr("incidents.alert_rules.threshold", tags={"type": "alert"})
+                            metrics.incr(
+                                "incidents.alert_rules.threshold.alert",
+                                tags={"type": detection_type},
```

Comment:
nit: might be good to specify the type so it's more clear in DD
```suggestion
                                tags={"detection_type": detection_type},
```